core/src/main/java/com/web3auth/core/types/LoginConfigItem.kt

```diff
@@ -5,7 +5,7 @@ data class LoginConfigItem(
     private var typeOfLogin: TypeOfLogin,
     private var name: String? = null,
     private var description: String? = null,
-    private var clientId: String,
+    private var clientId: String? = null,
```

Comment:
this is mandatory now

Reply:
its mandatory in Web3AuthOptions.

Reply:
this clientId is the OAuth client id.. like for auth0/firebase client id.
web3auth options client id is from our dashboard

Reply:
ok